### PR TITLE
Support testing against fuzz regressions

### PR DESCRIPTION
**Pull request by thomasvl**
* Set up directory for landing them.
* Seed it with the current failures.
* Update the fuzz helper script to run easily regressions also
* Revise the current workflow to run these regressions

### CODE REVIEW

File: FuzzTesting/README.md [hunk 1]
```diff
@@ -17,12 +20,27 @@ To build on linux:
...
+Then the binaries will be found in `.build/debug`.
 
 Note: You can also use `-c release` to build/test in release instead as that
 could find different issues.
 
-If/When issues are found, please consider adding them to
-`Tests/SwiftProtobufTests/Test_FuzzTests.swift`, not only does this make it
-easier to make sure things don't regres, but it also provides an easy way to
-debug them while working on the fix.
+In this directory you will also fine a `do_build.sh` script.  By default it
```
Comment (allevato) [inline FuzzTesting/README.md:28]: s/fine/find/

File: FuzzTesting/README.md [hunk 2]
```diff
@@ -17,12 +20,27 @@ To build on linux:
...
+
+1. Make sure you add a file to `FailCases` subdirectory so regressions can
+   easily be watched for. The GitHub workflow runs against there.
+
+2. Consider adding them to `Tests/SwiftProtobufTests/Test_FuzzTests.swift`, this
+   help in debugging while working on the failure, but also provides yet another way
+   to ensure things don't regress and if they do, it is much easier to immediately
+   debug the issue.
+
+A none about memory issues, the address sanitizer is enabled in the fuzz tests
```
Comment (allevato) [inline FuzzTesting/README.md:43]: s/none/note/

**Review by allevato — APPROVED** [review]

File: FuzzTesting/README.md (hunk 1 above)
Comment (thomasvl) [reply FuzzTesting/README.md]: done

File: FuzzTesting/README.md (hunk 2 above)
Comment (thomasvl) [reply FuzzTesting/README.md]: done

**Pull request merged**